platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less

```
@@ -630,7 +630,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       border-right: 5px solid @primaryColor ~'!important; /** orientation=rt */ ';
       padding: 10px 0 10px 10px ~'!important; /** orientation=lt */ ';
       padding: 10px 10px 10px 0 ~'!important; /** orientation=rt */ ';
-      p {
+      div {
```

```suggestion
      p, div {
```
we may need to keep `p` style as well to ensure retro-compatibility with old activities that used `p` in its content

---

Done!